scss/inc/_forms.scss

```diff
@@ -412,8 +412,6 @@ label {
     }
     input[type="radio"],
     input[type="checkbox"] {
-        visibility: hidden;
-        pointer-events: none;
```

Comment:
It's not so harmless:

![Screenshot_5](https://user-images.githubusercontent.com/38751932/147125751-07f287fc-c738-48bb-8a57-f2700c0f359b.png)

vs

![Screenshot_6](https://user-images.githubusercontent.com/38751932/147125755-cb7941fa-2bad-4663-88c7-d8850f6a7aa3.png)

Note that before `visibility:hidden`, it used offscreen `left` position: https://github.com/oat-sa/tao-core-ui-fe/commit/17b92c7f5098b9087c41d45267d900d9e6163013#diff-3bc1145cd4a253910fe0ef19c9411907c213932fc888152ff7dafefa924c126d
Maybe you should keep that solution (why did commit author remove it?).

Reply:
Actually solution which we discussed is a little bit wider. I just cut one property, because i found that it duplicates in other css rule. I tested it on some pages in my test and it works. I will revert this cut property and test it once more.

About the testing you are right. Typically we do not test technical tickets, but it is not a rule, it is because no way to test in QA. Here other situation. I ask our QA, which way they prefer: separate ticket or do it in terms of this.

Reply:
I found one more issue in authoring and fix this, but in any case i will create separate ticket and will continue on it.